### PR TITLE
updated CSS, updated quotes, updated maps on each page to have the correct lat long, etc

### PR DESCRIPTION
**Pull request by semenza2**
Screen Shot 2019-07-06 at 2 23 34 PM
this is what the homepage looks like now

### CODE REVIEW

File: src/main/webapp/js/map-loader.js [hunk 1]
```diff
@@ -45,6 +45,34 @@ function createMap(){
...
+	})
+	fetchMarkers();
+}
+
+/**
+* Venice main function
+*/
+function createVeniceMap(){
+	map = new google.maps.Map(document.getElementById('map'), {
+	center: {lat: 45.4408, lng: 12.3155},
```
Comment (dndquynh) [inline src/main/webapp/js/map-loader.js]: The code is repeated so maybe create one common function and pass the lat and lng as the params to remove redundant code.

File: src/main/webapp/user-page.html [hunk 2]
```diff
@@ -30,10 +29,12 @@
   </head>
   <body onload="buildUI();fetchBlobstoreUrlAndShowForm();">
     <div id="content">
-			<nav id="navigation">
-      	<ul>
+			<nav>
+      	<ul id="navigation">
         	<li><a href="/">Home</a></li>
+					<li><a href="/amsterdam.html">Amsterdam</a></li>
         	<li><a href="/paris.html">Paris</a></li>
```
Comment (dndquynh) [inline src/main/webapp/user-page.html:36]: nit: indentation seems off here

File: src/main/webapp/js/map-loader.js (hunk 1 above)
Comment (semenza2) [reply src/main/webapp/js/map-loader.js]: yes good idea, fixed

**Review by dndquynh — APPROVED** [review]
LGTM

**Pull request merged**